app/controllers/volunteer_applications_controller.rb

```diff
@@ -7,6 +7,8 @@ def create
     @volunteer_application =
       VolunteerApplication.new(volunteer_application_params)
     if @volunteer_application.save
+      NewVolunteerApplicationMailer.volunteer_notification(
```

Comment:
Looks like this is from https://github.com/JessSumner/any_rat_rescue/pull/14 would you mind rebasing once that's merged?